frontend/src/App.js

```diff
@@ -79,6 +81,8 @@ const App = () => (
           {/*Routes that only logged in Users can access*/}
           <UserRoute path="/useronly" component={Debug} />
           <UserRoute path="/account" component={AccountPage} />
+          <UserRoute path="/followers/:id" component={FollowersPage} />
```

Comment:
If we are only going to display the followers and followings on the Account page, I don't think we need routes.